src/Arcus.Security.Core/Exceptions/SecretNotFoundException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Arcus.Security.Core.Exceptions
+{
+    /// <summary>
+    /// Exception, thrown when no secret was found, using the given name.
+    /// </summary>
+    public class SecretNotFoundException : Exception
+    {
+        public string Name { get;  }
```

Comment:
I should look into the formatting guidelines of the Arcus project, but I think there's something about having a blank line between properties and members + having them below the ctor(s).

Not sure though; not blocking for me.

Reply:
I changed it according to your request, but I'll leave it up to you and @tomkerkhove to detail it out.